JPF/utils.py

```diff
@@ -104,6 +104,7 @@ def hpd_csv2sql(description, input_csv_url, sep_char,\
         max_col_len - length to truncate cells in truncate_columns to
         date_format - expected format of dates in the table
         csv_chunk_size - if defined, the number of rows to process from the CSV at a time.
+        set_default_primary_key - if True, adds an ID column to serve as primary key.
```

Comment:
Presumably, there should be an arg allowing callers to specify a column name to use as the primary key (ucbbl in this case), but I'm not sure what the SQL syntax is for the ALTER TABLES call below.